README.md

```diff
@@ -215,6 +215,39 @@ If your Sinatra app consists of subprojects and you want to capture errors
 separately for each subproject, make sure to [configure it
 accordingly](#configuring-individual-notifier-for-each-subproject).
 
+### Hanami
+
+To use Airbrake with specific [hanami](http://hanamirb.org) app you need
+to create initializer file and and `use` our Rack middleware in app
+configuration.
+
+```ruby
+# config/initializers/airbrake.rb
+require 'airbrake'
+
+Airbrake.configure do |c|
+  c.project_id = 113743
+  c.project_key = 'fd04e13d806a90f96614ad8e529b2822'
+
+  # Display debug output.
+  c.logger.level = Logger::DEBUG
+end
+
+# apps/web/application.rb
+module Web
+  class Application < Hanami::Application
+    configure do
+      # ...
+      middleware.use use Airbrake::Rack::Middleware
+      # ...
+    end
+  end
+end
+```
+
+If you want to use airbrake for full hanami stack you can use Rack integration.
+For this you need to add Rack middleware to `config.ru` file.
```

Comment:
I would also shorten this sentence:

> To use Airbrake for full hanami stack, add our \<a\>Rack integration\</a\> to the `config.ru` file (instead of `application.rb`).